app/javascript/miq-redux/store.ts

```diff
@@ -0,0 +1,19 @@
+import { createStore, applyMiddleware, compose, Store} from 'redux';
+import {rootReducer} from './reducer';
+
+import {AppState} from '../packs/miq-redux';
+
+const composeEnhancers = window['.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__'] || compose;
```

Comment:
We were planning ahead and added support for middlewares as well, and based on their docs we should use [advanced-store-setup](https://github.com/zalmoxisus/redux-devtools-extension#user-content-12-advanced-store-setup) with middlewares.